matroska_tags.xml

```diff
@@ -276,10 +276,10 @@
       <description lang="en">This is a binary dump of the TOC of the CDROM that this item was taken from. This holds the same information as the MCDI in ID3.</description>
     </tag>
     <tag name="ISBN" class="Identifiers" type="UTF-8">
-      <description lang="en">[International Standard Book Number](http://www.isbn-international.org/)</description>
+      <description lang="en">[International Standard Book Number](https://www.isbn-international.org/)</description>
     </tag>
     <tag name="BARCODE" class="Identifiers" type="UTF-8">
-      <description lang="en">[EAN-13](http://www.ean-int.org/) (European Article Numbering) or [UPC-A](http://www.uc-council.org/) (Universal Product Code) bar code identifier</description>
+      <description lang="en">[EAN-13](https://www.gs1.org/) (European Article Numbering) or [UPC-A](http://www.uc-council.org/) (Universal Product Code) bar code identifier</description>
```

Comment:
I would prefer this less generic URL https://www.gs1.org/standards/barcodes/ean-upc
The top site doesn't even look like it's dealing with barcodes.

Reply:
done